packages/plugins/rubygem-smart_proxy_dynflow_core/rubygem-smart_proxy_dynflow_core.spec

```diff
@@ -119,6 +123,9 @@ install -Dp -m0644 %{SOURCE1} %{buildroot}%{root_sysconfdir}/logrotate.d/%{name}
 %doc %{gem_instdir}/LICENSE
 
 %changelog
+* Tue Jan 07 2020 Eric D. Helms <ericdhelms@gmail.com> - 0.2.4-2
+- Add obsoletes on non-SCL version of the package.
```

Comment:
I've added this so that Jenkins is happy. Technically we don't have to rebuild the *_core packages, as they are in the SCL already.